pkg/metrics/source/stats_reporter.go

```diff
@@ -62,6 +63,7 @@ var (
 type ReportArgs struct {
 	Namespace     string
 	EventType     string
+	EventScheme   string
```

Comment:
Same as with adapter, I'm not sure if we are reporting metrics or not for this - I only know for sure about MTChannelBasedBroker and InMemoryChannel. cc @pierDipi for more info